src/zcl/definition/cluster.ts

```diff
@@ -2087,6 +2089,15 @@ const Cluster: {
                     {name: 'transtime', type: DataType.uint8},
                 ],
             },
+            moveToHueAndSaturationBrightness: {
```

Comment:
I've checked the ZCL: https://github.com/Koenkk/zigbee-herdsman/blob/master/docs/Zigbee%20Cluster%20Library%20Specification%20v7.pdf and command 6 is the `moveToHueAndSaturation` and does not provide a brigthness:

![image](https://user-images.githubusercontent.com/2892853/96007445-c58c4e80-0e3e-11eb-9cc4-13c8913615c6.png)

Reply:
Yes I know, but unfortunately my TuYa LEDs don't seem follow the standard for whatever reason. Please see attached an image of a packet captured via Wireshark.
The payload contains FF which is the brightness in the end.

<img width="400" alt="wireshark" src="https://user-images.githubusercontent.com/24277620/96045039-b246a680-0e71-11eb-82e1-b161f64dcd5e.PNG">


Hue: 0x55 = 85
Saturation: 0xFE = 254
Transition time: 0x0000 = 0
Brightness: 0xFF = 255

Reply:
In that case change `moveToHueAndSaturationBrightness` to `tuyaMoveToHueAndSaturationBrightness`